src/cmd-build-with-buildah

```diff
@@ -161,8 +161,13 @@ build_with_buildah() {
     fi
 
     if [ -d overrides ]; then
-        if [[ -n $(ls overrides/rpm/*.rpm 2> /dev/null) ]]; then
-            (cd overrides/rpm && rm -rf .repodata && createrepo_c .)
```

Comment:
Hmm, I don't understand what exactly you were hitting.

Actually, is the bug here just `s/.repodata/repodata`? Though this was cargo-culted from https://github.com/coreos/coreos-assembler/blob/6e863df1d683a19182c9294d27b5856b018d50e2/src/cmdlib.sh#L447.

Reply:
This is outdated now and already handled in another commit that's already upstream.